### PR TITLE
Fix body parameters

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: src/plugins/azgenerator/CodeModelAzImpl.ts [hunk 1]
```diff
@@ -263,7 +267,7 @@ export class CodeModelCliImpl implements CodeModelAz
 
     public get Command_Name(): string
     {
-        this.session.message({Channel:Channel.Warning, Text: "operationGroupIndex: " + this.currentOperationGroupIndex + " operationIndex: " + this.currentOperationIndex});
+        //this.session.message({Channel:Channel.Warning, Text: "operationGroupIndex: " + this.currentOperationGroupIndex + " operationIndex: " + this.currentOperationIndex});
         return this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].language['az'].command;
```
Comment (RodgeFu) [inline src/plugins/azgenerator/CodeModelAzImpl.ts]: Please remove if it's not needed, same for other commented code. thanks.

File: src/plugins/azgenerator/TemplateAzureCliCustom.ts [hunk 2]
```diff
@@ -128,72 +133,68 @@ function GetCommandBody(model: CodeModelAz, required: boolean, needUpdate: boole
...
+                        let body = model.MethodParameter;
+
+                        while(model.SelectNextMethodParameter()) {
+                            let access = "    " + bodyName;
+                            let param = model.MethodParameter;
+                            let oriParam = (param['originalParameter']);
+                            if(oriParam == body) {
+                                if(param['pathToProperty']?.length == 1) {
+                                    let pathParam = param['pathToProperty'][0];
+                                    access += ".setdefault('" + pathParam.language['python'].name + "', {})";
```
Comment (RodgeFu) [inline src/plugins/azgenerator/TemplateAzureCliCustom.ts]: >".setdefault('" + pathParam.language['python'].name + "', {})" [](start = 46, length = 62)

from curious, any reason not to use `` ?

File: src/plugins/azgenerator/CodeModelAzImpl.ts (hunk 1 above)
Comment (qiaozha) [reply src/plugins/azgenerator/CodeModelAzImpl.ts]: removed.

File: src/plugins/azgenerator/TemplateAzureCliCustom.ts (hunk 2 above)
Comment (qiaozha) [reply src/plugins/azgenerator/TemplateAzureCliCustom.ts]: This is copied from the original custom template. I changed it now

**Pull request merged**